apps/src/aichat/views/aichatView.module.scss

```diff
@@ -33,6 +33,7 @@
 
   .presentationArea {
     width: 50%;
-    height: 100%;
+    height: 90%;
```

Comment:
This is to avoid the language selection footer on the bottom lefthand side of screen when multiple sections are expanded.